Example/Keys.plist

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There's no key here. It's credentials, or client secrets. The file name is misleading.